setup.php

```diff
@@ -55,6 +55,8 @@ function plugin_init_uninstall() {
       $UNINSTALL_DIRECT_CONNECTIONS_TYPE  = ['Monitor', 'Peripheral', 'Phone',
                                              'Printer'];
 
+      Conf::$defaults['plugin_uninstall_stale_agents_uninstall'] = 0;
+
```

Comment:
Seems that it is never used.

Reply:
It at least suppresses the warning/error message when saving that says one or more config options are unknown.